src/naksu/ui.go

```diff
@@ -92,7 +93,14 @@ func createMainWindowElements() {
 	comboboxLang.Append("Suomeksi")
 	comboboxLang.Append("På svenska")
 	comboboxLang.Append("In English")
-	comboboxLang.SetSelected(0)
+	switch config.GetLanguage() {
+	case "fi":
+		comboboxLang.SetSelected(0)
+	case "sv":
+		comboboxLang.SetSelected(1)
+	case "en":
+		comboboxLang.SetSelected(1)
```

Comment:
Tässä pitäis varmaan olla `default` tai ehkä mieluummin `config`-moduulin pitäisi pitää huoli että ulos tulee vain sallitu arvo

Reply:
käytännössä arvoksi jää tuossa tapauksessa alustettu default (suomi). sen sijaan indeksihän tuossa on väärin